### PR TITLE
Add The Sims 3

### PR DESCRIPTION
**Pull request by Kinsiinoo**
Adds The Sims 3.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by AuraOfTheDawn** [comment]
US Boxart is standard according to the contributing.md guide, can you find a US release boxart shot please? Other then that, looks good to me.

File: games/the-sims-3/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "The Sims 3"
+description = "The Sims 3 is a strategy / life simulation game developed by The Sims Studio and published by Electronic Arts. Create Sims, the characters unique to The Sims franchise, with unique personalities, fulfill their desires and control their lives within a living neighborhood. Unlock new Karma Powers with Karma Points and unleash amazing effects on your Sims. As you guide your Sims through life, build relationships and experience different careers like Criminal, Athlete or Musician to unlock new items, outfits and various rewards."
```
Comment (CodingKoopa) [inline games/the-sims-3/game.dat]: This should be written less as trying to sell the game, and more from a neutral perspective, like [Wikipedia](https://en.wikipedia.org/wiki/The_Sims_3).

File: games/the-sims-3/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+needs_system_files = false
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000033D00"
+region = "eur"
```
Comment (CodingKoopa) [inline games/the-sims-3/game.dat]: USA release first.

**Comment by Kinsiinoo** [comment]
- Added usa region primary for praising
- Implemented the wrong version (pc) wikipedia description
😄 😬

File: games/the-sims-3/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
+title = "The Sims 3"
+description = "The Sims 3 is a strategy / life simulation game developed by The Sims Studio and published by Electronic Arts. The Sims 3 is built upon the same concept as its predecessors. Players control their own Sims' activities and relationships in a manner similar to real life. The game play is open-ended and does not have a defined goal. Challenges occur randomly based on aspects of each Sim's lifestyle, such as relationships, skills and job. Career opportunities such as working overtime or completing special tasks can yield a pay raise, cash bonus, or relationship boost. Skill opportunities are requests by neighbors or community members for Sims to solve problems using their acquired skills for cash or relationship rewards. If the opportunity is connected to a Sim's school, the reward may be increased school performance."
```
Comment (Leo626) [inline games/the-sims-3/game.dat]: Description could be cut down. I say the first 3 sentences are enough for the description.

**Pull request merged**